web/app/js/components/MetricsTable.test.js

```diff
@@ -26,7 +26,7 @@ describe('Tests for <MetricsTable>', () => {
 
     expect(table).toBeDefined();
     expect(table.props().tableRows).toHaveLength(1);
-    expect(table.props().tableColumns).toHaveLength(8);
+    expect(table.props().tableColumns).toHaveLength(9);
```

Comment:
Should there be a negative test added? eg. to test when `jaeger=false`?

Reply:
Actually I made the jaeger flag optional now, and added additional test for `jaeger=true`